redisinsight/api/src/modules/browser/services/z-set-business/z-set-business.service.ts

```diff
@@ -406,11 +407,13 @@ export class ZSetBusinessService {
     const result: ZSetMemberDto[] = [];
     while (reply.length) {
       const member = reply.splice(0, 2);
+      const score = isNaN(parseFloat(member[1])) ? String(member[1]) : parseFloat(member[1]);
       result.push(plainToClass(ZSetMemberDto, {
```

Comment:
we are using "plainToClass" here. Maybe it is better to create own transformer and `isNaN(parseFloat(member[1])) ? String(member[1]) : parseFloat(member[1]);` will be done automatically inside?
P.S. See comment above